src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        action="/"
+        method="get"
+        data-qa="big"
+        class="big-form"
+      >
+        <img src="./images/Search.svg" alt="search-img" class="big-image">
+        <label for="big-search"></label>
+        <input
+          type="text"
+          class="search big--search"
+          id="big-search"
+          name="big-search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
```

Comment:
it doesn't make much sense when stacked this way - label is not functional.
And this stack even has a downside - when clicking the `magnifying glass` icon - you're not getting focused on an input.
So, better approach would be to nest both `<img` and `<input` inside `<label` -> then clicking icon would make input focused:
```suggestion
        <label for="big-search">
          <img src="./images/Search.svg" alt="search-img" class="big-image">
          <input
            type="text"
            class="search big--search"
            id="big-search"
            name="big-search"
            placeholder="Try &quot;Los Angeles&quot;"
            data-qa="keypress"
          >
        </label>
```